ChaosMod/Effects/db/Player/PlayerKeepRunning.cpp

```diff
@@ -42,4 +42,4 @@ REGISTER_EFFECT(nullptr, nullptr, OnTick, EffectInfo
 		.IsTimed = true,
 		.IsShortDuration = true
 	}
-);
\ No newline at end of file
+);
```

Comment:
It's back again 😄

Reply:
Arghh…. How do I remove this?

Reply:
Are you editing it using the web interface? It seems to automatically add a newline to the end of the file.

Reply:
Yeah, I do it for small edits where it’s not necessary to open up my ide

Reply:
Fixed it